aiida/transports/plugins/ssh.py

```diff
@@ -1309,6 +1337,15 @@ def gotocomputer_command(self, remotedir):
         )
         return cmd
 
+    def symlink_internal(self, source, dest):
```

Comment:
Since this is just an internal method, maybe a better name would be simply `_symlink`?

Reply:
Indeed. Done in 81319e4